pkg/solana/client/multinode/client_test.go

```diff
@@ -94,6 +94,31 @@ func TestMultiNodeClient_HeadSubscriptions(t *testing.T) {
 			t.Fatal("failed to receive finalized head: ", ctx.Err())
 		}
 	})
+
+	t.Run("Remove Subscription on Unsubscribe", func(t *testing.T) {
```

Comment:
Could you also add a test to verify that UnsubscribeAllExecept won't remove & unsubscribe passed sub?

Reply:
Yes, I added a test for `UnsubscribeAllExcept`